packages/components/src/DataTable/DataTable.mdx

```diff
@@ -0,0 +1,224 @@
+        isAlive: "Yes",
+      },
+      {
+        name: "Tommen",
+        house: "Baratheon",
+        region: "Stormlands",
+        sigil: "Black Stag",
+        isAlive: "Yes",
+      },
+    ]}
+    pagination={{ manualPagination: false, itemsPerPage: [10, 20, 30] }}
+    sorting={{ manualSorting: false }}
+    columns={[
+      {
+        accessorKey: "name",
+        cell: info => <span>{info.getValue()}</span>,
+      },
+      {
+        accessorKey: "house",
+        cell: info => info.getValue(),
+      },
+      {
+        accessorKey: "region",
+        cell: info => info.getValue(),
+      },
+      {
+        accessorKey: "sigil",
+        cell: info => info.getValue(),
+      },
+      {
+        accessorKey: "isAlive",
+        cell: info => info.getValue(),
+      },
+    ]}
+  />
+</Playground>
+
+<!--
+  INTERFACE: This is for the proposal stage of the component
+
+  Provide an example of what the component looks like in code. How would you use
+  it from another React component? This should consist primarily of code blocks.
+  This section can be deleted once the component is built, when the Playground
+  will demonstrate this at a higher fidelity.
+-->
+
+## Props
+
+<Props of={DataTable} />
+
+<!--
+  It is not necessary to provide an example of each prop. Use usage examples to
+  highlight key features of a component or particular considerations.
+
+  See `Button.mdx` for a good example of this.
+-->
+
+## Usage Guidelines
+
+The `<DataTable>` will... _Add a brief description of the component_
+
+<!--
+  What is the design purpose of this component? How do its responsibilities
+  relate to other components? What should this component not be used for? What
+  are some related components in Atlantis? Try to describe this component not
+  in terms of what it looks like or what elements make it up, but what function
+  it performs or how it enables a user to perform tasks (e.g. “Buttons allow users
+  to initiate, complete, and reverse actions.” vs “Buttons are rectangular elements
+  that have a green background and white text”).
+-->
+
+## Content Guidelines
+
+<!--
+  What type of content does this component present (documents, text, images, other components)?
+  What is the recommended, or expected, amount of content? What is the components’ behaviour when
+  the provided content diverges from what is expected (image aspect ratio, amount of text, video
+  vs audio, filesize)? Are there any important references to the [Product Vocabulary](https://atlantis.getjobber.com/guides/product-vocabulary)
+  that should be made regarding this component? What happens when content is unavailable or unreliable
+  (poor connection, empty states, flaky GPS data?)
+-->
+
+## Accessibility
+
+<!--
+  Describe the accessibility concerns for the component. How does it handle touch vs cursor vs
+  keyboard events? Should it capture input? What should a screen reader see when it's focuse?
+-->
+
+## Responsiveness
+
+<!--
```

Comment:
Assuming this is all still WIP

Reply:
Oh yeah, this is still a draft in regard to code structure, docs,  and whatnot.